flagsmith/flagsmith.py

```diff
@@ -102,17 +105,11 @@ def get_value(self, feature_name, identity=None):
 
         data = self._get_flags_response(feature_name, identity)
 
-        if data:
-            # using new endpoints means that the flags come back in a list, filter this for the one we want and
-            # return it's value
-            if data.get("flags"):
```

Comment:
@matthewelwell  @dabeeeenster  The current api only returns the flags asked and not the list, can we safely remove this?